src/ShipCpanel.cpp

```diff
@@ -280,12 +287,12 @@ void ShipCpanel::Update()
 	int timeAccel = m_game->GetTimeAccel();
 	int requested = m_game->GetRequestedTimeAccel();
 
-	for (int i=0; i<6; i++) {
+	for (int i=0; i<7; i++) {
```

Comment:
>  it would be nice if this and other hardcoded numbers used the values from the enum instead of '5', '7' etc. Would you mind making that change?

Then please do it as a separate commit (or PR) so we can cherry-pick it if this PR isn't merged.